packages/material-ui-lab/src/Alert/Alert.js

```diff
@@ -1,296 +1,22 @@
-      border: `1px solid ${theme.palette.warning.main}`,
-      '& $icon': {
-        color: theme.palette.warning.main,
-      },
-    },
-    /* Styles applied to the root element if `variant="outlined"` and `color="error"`. */
-    outlinedError: {
-      color: getColor(theme.palette.error.main, 0.6),
-      border: `1px solid ${theme.palette.error.main}`,
-      '& $icon': {
-        color: theme.palette.error.main,
-      },
-    },
-    /* Styles applied to the root element if `variant="filled"` and `color="success"`. */
-    filledSuccess: {
-      color: '#fff',
-      fontWeight: theme.typography.fontWeightMedium,
-      backgroundColor: theme.palette.success.main,
-    },
-    /* Styles applied to the root element if `variant="filled"` and `color="info"`. */
-    filledInfo: {
-      color: '#fff',
-      fontWeight: theme.typography.fontWeightMedium,
-      backgroundColor: theme.palette.info.main,
-    },
-    /* Styles applied to the root element if `variant="filled"` and `color="warning"`. */
-    filledWarning: {
-      color: '#fff',
-      fontWeight: theme.typography.fontWeightMedium,
-      backgroundColor: theme.palette.warning.main,
-    },
-    /* Styles applied to the root element if `variant="filled"` and `color="error"`. */
-    filledError: {
-      color: '#fff',
-      fontWeight: theme.typography.fontWeightMedium,
-      backgroundColor: theme.palette.error.main,
-    },
-    /* Styles applied to the icon wrapper element. */
-    icon: {
-      marginRight: 12,
-      padding: '7px 0',
-      display: 'flex',
-      fontSize: 22,
-      opacity: 0.9,
-    },
-    /* Styles applied to the message wrapper element. */
-    message: {
-      padding: '8px 0',
-    },
-    /* Styles applied to the action wrapper element if `action` is provided. */
-    action: {
-      display: 'flex',
-      alignItems: 'center',
-      marginLeft: 'auto',
-      paddingLeft: 16,
-      marginRight: -8,
-    },
-  };
-};
-
-const defaultIconMapping = {
-  success: <SuccessOutlinedIcon fontSize="inherit" />,
-  warning: <ReportProblemOutlinedIcon fontSize="inherit" />,
-  error: <ErrorOutlineIcon fontSize="inherit" />,
-  info: <InfoOutlinedIcon fontSize="inherit" />,
-};
-
-const Alert = React.forwardRef(function Alert(props, ref) {
-  const {
-    action,
-    children,
-    classes,
-    className,
-    closeText = 'Close',
-    color,
-    icon,
-    iconMapping = defaultIconMapping,
-    onClose,
-    role = 'alert',
-    severity = 'success',
-    variant = 'standard',
-    ...other
-  } = props;
-
-  const themeVariantsClasses = useThemeVariants(
-    {
-      ...props,
-      closeText,
-      iconMapping,
-      role,
-      severity,
-      variant,
-    },
-    'MuiAlert',
+import React from 'react';
+import warning from 'warning';
+import Alert from '@material-ui/core/Alert';
+
+export default React.forwardRef(function DeprecatedAlert(props, ref) {
+  let warnOnce = false;
```

Comment:
To match ExpansionPanel.js in v4.

```suggestion
let warnedOnce = false;
 
export default React.forwardRef(function DeprecatedAlert(props, ref) {
```